themes/default/history.js

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from 'history'
+
+export default createBrowserHistory()
```

Comment:
This will probably error during SSR. Maybe add `if (typeof window !== 'undefined')`?

Reply:
yeah was about to test that

Reply:
https://app.netlify.com/sites/gitdocs/deploys/5af36da70733d51a61e028a9